tests/e2e/search-product.e2e.test.js

```diff
@@ -0,0 +1,69 @@
+const { test, expect } = require('@playwright/test')
+
+const {
+  createUser,
+  createProduct,
+  getUserBody,
+  getProductBody,
+  login,
+} = require('../../lib/helpers')
+
+test.describe.parallel('Search product', () => {
+  test.beforeEach(async ({ page }) => {
+    const user = getUserBody()
+
+    await page.goto('/cadastrarusuarios')
+
+    await createUser(
+      page,
+      user.nome,
+      user.email,
+      user.password
+    )
+
+    await page.waitForNavigation()
+    await expect(page).toHaveURL('/admin/home')
+
+    await page.goto('/login')
+
+    await login(
+      page,
+      user.email,
+      user.password
+    )
+
+    await expect(page).toHaveURL('/admin/home')
```

Comment:
@uLucasFraga Let's simply use the helper function here. We can also update the `create-product` test.

```suggestion
    const user = getUserBody()
    await openHomepage(page, user)
```